v2/bolt_store.go

```diff
@@ -0,0 +1,268 @@
+package raftboltdb
+
+import (
+	"errors"
+
+	"go.etcd.io/bbolt"
```

Comment:
I'mma just point out all the changes to bbolt so it's a bit easier to read, so here's 1 (there are not many)